### ISSUE TITLE
support for `redrawOnResize` func

### ISSUE THREAD

**Issue by nikhil478** [agilgur5/react-signature-canvas#94]
available in `react-signature-pad`. if available can u please provide some sample code !

**Comment by agilgur5** [comment]
Per the description of this component, `react-signature-canvas` is a fork of [`react-signature-pad`](https://github.com/blackjk3/react-signature-pad) already and is heavily updated with numerous other features.
`react-signature-pad` has also not had a commit in 5+ years.

So I'm not sure what you're referencing, but it does not have any such `redrawOnResize` function.

Please recognize that open-source maintainers are unpaid volunteers -- filing an issue is a request for a maintainer's free time. If you are making such a request for another person's time, please make sure it is, at the minimum, detailed and specific, in order to be as respectful of that person's time as possible.
With that in mind, a one sentence issue without a description, with no links, and an invalid reference is likely not very respectful by those standards.

**Comment by nikhil478** [comment]
**it was actually react wrapper i used it before using signature-canvas and i think it was signature pad**
Sorry for that i m just new to community ! i can understand and apologized for same !

link of the repo: https://github.com/michaeldzjap/react-signature-pad-wrapper

![Screenshot (475)](https://user-images.githubusercontent.com/67046318/194875699-b20956bf-4967-4369-a1c7-812097e97a60.png)

can u  provide support of redrawOnResize in future release! as redrawOnResize currently NA on signature-canvas 

basically what i need is height and width of signature canvas adjust according to size of window without loss of data. this feature is available in react-signature-pad-wrapper as redrawOnResize !
